src/vehicle.h

```diff
@@ -788,45 +779,45 @@ class vehicle : public JsonSerializer, public JsonDeserializer
 
     void shed_loose_parts();
 
-    // Gets range of part p if it's a turret
-    int get_turret_range( int p );
+    /**
+     * @name Vehicle turrets
+     *
+     *@{*/
+
+    /** Get all vehicle turrets (excluding any that are destroyed) */
+    std::vector<vehicle_part *> turrets();
 
-    // Returns the number of shots this turret could make with current ammo/gas/batteries/etc.
-    // Does not handle tags like FIRE_100
-    class turret_ammo_data;
-    turret_ammo_data turret_has_ammo( int p ) const;
+    /** Get all vehicle turrets loaded and ready to fire at @ref pos */
+    std::vector<vehicle_part *> turrets( const tripoint &pos );
```

Comment:
Ambiguous: are the turrets at `pos` or are they firing at `pos`? Both are feasible here.